2_phase_minting_schedule/main.tex

```diff
@@ -4,61 +4,62 @@
+We propose a two-phase subsidy model to bootstrap the NuCypher network, the first phase comprising a \textit{stable issuance}, in which approximately 366 million new tokens are minted and distributed to active (staking) node operators every year in the first phase, set to 5 years. This is equivalent to expanding the supply by 1,002,509 tokens in each of the first 1825 periods (days). In the second phase, commencing on the first period of the fifth year, fewer tokens will be minted in each successive period, such that the daily minting figure decays exponentially with a half-life of 2 years. Over the course of the network's lifetime – as $t$ approaches $\infty$ – the circulating supply will approach a maximum of roughly 3.885 billion tokens. 
 
 \begin{figure}[h!]
-    \includegraphics[width=\textwidth]{Two_phase_inflation.png}
-    \caption{Illustration comparing the previous draft and two-phase model over the first 6 years of the network's lifetime.}
+    \includegraphics[width=\textwidth]{Two_phase_model.png}
+    \caption{Illustration comparing a two-phase model with a common subsidy model, over the first 6 years of the network's lifetime. Note that the y-axis refers to an absolute figure, not a rate.}
     \label{fig:tp}
 \end{figure}
 
-An alternative to the previous model is proposed here; a static (unchanging) inflation rate of \textbf{36.6\%} for the first 5 years, which implies the minting of approximately 366 million new tokens per year. From day one of the fifth year, the inflation rate will decay continuously with a half-life of 2 years, so fewer tokens are minted in each successive period. Over the course of the network's lifetime, a similar number of tokens will be minted as in the previous draft – i.e. as $t$ approaches $\infty$, the circulating supply will reach approximately 3.885 billion tokens. The next two sections provide a detailed rationale for the new, `two-phase' model, and the final section derives a new genesis inflation figure and presents a new minting schedule.
+Monetary policy designed for decentralized networks often make reference to a so-called `inflation rate'. In this model, phase one sees an \textbf{identical sum of tokens} minted in each period, so the effective `rate' at which the circulating supply grows decreases period-to-period (and year-to-year). In phase two, the number of tokens minted per day also decreases, which accelerates the decline in the supply growth rate. To avoid misleading terminology, the number of total tokens minted per day shall be hereafter referred to as the \textit{issuance}. Note that the effective growth rate of any given stake also decreases over time, the extent to which is subject to discretionary factors such as the percentage of tokens re-staked by the node operator. 
```

Comment:
```suggestion
Monetary policy designed for decentralized networks often make reference to a so-called `inflation rate'. In this model, phase one sees an \textbf{constant amount of tokens} minted in each period, so the effective `rate' at which the circulating supply grows decreases period-to-period (and year-to-year). In phase two, the number of tokens minted per day also decreases, which accelerates the decline in the supply growth rate. To avoid misleading terminology, the number of total tokens minted per day shall be hereafter referred to as the \textit{issuance}. Note that the effective growth rate of any given stake also decreases over time, the extent to which is subject to discretionary factors such as the percentage of tokens re-staked by the node operator. 
```